src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -73,6 +73,88 @@ def _core_registry_scenario(self, registry_name, resource_group, location,
         # test acr delete
         self.cmd('acr delete -n {} -g {}'.format(registry_name, resource_group))
 
+    @ResourceGroupPreparer
+    def test_acr_image_import(self, resource_group):
+        '''There are six test cases in the function, first of which should be executed in the 'ACR - CI' subscription and the rest are in 'ACR - Dev' subscription.
+        The test cases assume that there are source registry, source image and destination registry.'''
+
+        self.cmd("account set -s 'ACR - CI'")
+        #Case 1: Import image from one regsitry to another where both registries belong to different subscriptions
+        self.kwargs.update({
+            'registry_name': 'yugongTest2',
+            'r_id': '/subscriptions/dfb63c8c-7c89-4ef8-af13-75c1d873c895/resourceGroups/yugong/providers/Microsoft.ContainerRegistry/registries/yugongeast',
```

Comment:
Pre-create two registries, and push a source image, initialize all parameters here:
```
'r_id_same_sub': '<id>'
'r_id_different_sub': '<id>'
'target_tag1': 'targetrepository:targettag'
...
```
Use some generic names rather than your names.
Add a comment here mentioning something like we require the existence of source image/registry.

Call `self.kwargs.update` only once in this test.